server_test.go

```diff
@@ -50,17 +52,26 @@ func AnotherHelloServer(w ResponseWriter, req *Msg) {
 }
 
 func RunLocalUDPServer(laddr string) (*Server, string, error) {
-	server, l, _, err := RunLocalUDPServerWithFinChan(laddr)
+	server, l, _, err := RunLocalUDPServerWithFinChan(laddr, 0)
```

Comment:
Probably makes sense to make a new RunLocalUDPserverWithFinChan function that sets the rdtimeout (or set it after you created it?)